npm/examples.js

```diff
@@ -0,0 +1,6 @@
+const log = require('bestikk-log');
+const Builder = require('./builder.js');
+const builder = new Builder();
+
+log.task('examples');
+builder.examples();
```

Comment:
As mentioned in https://github.com/asciidoctor/asciidoctor-reveal.js/pull/156#issuecomment-335613495 I think we should use an "examples" task to demonstrate how to use the converter in JavaScript.

In the Asciidoctor.js repository, I have one example with a textarea on the left of the page and the rendered Reveal.js presentation on the right of the page.

Reply:
Not sure I understand what you mean. We have it in the documentation already. Adding an example  would mean maintaining two copies of the same code and instructions.

Reply:
Alright, I thought the purpose of the examples task was to provide a "live" example.